vault/pkg.plugin-core.dev.cook.md

```diff
@@ -209,3 +209,10 @@ if (PickerUtilsV2.isStringInputEmpty(out)) return;
 - A lot of the times you need to offset the line count of the file content of a markdown file so that it takes into account of the prepended frontmatter.
 - You can use the utility method `RemarkUtils.getNodePositionPastFrontmatter` to get the position where the first non-frontmatter text starts.
 - The return value of this can be converted to VSCode positions with `VSCodeUtils.point2VSCodePosition`
+
+### Use DevTrigger Command for development
+[DevTriggerCommand](https://github.com/dendronhq/dendron/blob/master/packages/plugin-core/src/commands/DevTriggerCommand.ts) is available to be invoked from Command Palette while in Development Mode.
```

Comment:
can you use a file link for the code? `[[../packages/plugin-core/src/commands/DevTriggerCommand.ts]]` 

i'll be sending an async about this soon but with kaan's integration changes, we're moving to getting everyone using file links